### PR TITLE
chore: fix make docker build xxx

### PR DESCRIPTION
**Pull request by j4ckstraw**
### Ⅱ. Does this pull request fix one issue?

fix https://github.com/koordinator-sh/koordinator/issues/1945

### V. Checklist

- [ ] I have written necessary docs and comments
- [ ] I have added necessary unit tests and integration tests
- [ ] All checks passed in `make test`

### CODE REVIEW

File: docker/koordlet.dockerfile [hunk 1]
```diff
@@ -7,6 +7,7 @@ ENV VERSION $VERSION
 ENV GOOS linux
 ENV GOARCH $TARGETARCH
 
+RUN sed -i 's#deb.debian.org#ftp.cn.debian.org#' /etc/apt/sources.list
```
Comment (j4ckstraw) [inline docker/koordlet.dockerfile:10]: What do you think?

Comment (j4ckstraw) [reply docker/koordlet.dockerfile:10]: refer https://www.debian.org/mirror/list

Comment (saintube) [reply docker/koordlet.dockerfile:10]: This modification is a localized improvement and may not help the GitHub workflow.

Comment (j4ckstraw) [reply docker/koordlet.dockerfile]: OK, removed.

**Review by hormes — APPROVED** [review]

**Pull request merged**